src/wesnoth.cpp

```diff
@@ -941,7 +941,7 @@ static void restart_process(const std::vector<std::string>& commandline)
 #define error_exit(res) return res
 #endif
 
-#ifdef __APPLE__
+#ifdef __IPHONEOS__
```

Comment:
I build and run both Mac and iOS version. Though, my build is probably unlike the current Mac port: I use CMake and MacPorts' libraries, while mainline obviously uses XCode and I don't event know what dependency management.
I don't have to commit this one, though.